manifests/t/Tgstation/Server/5.14.0/Tgstation.Server.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Tgstation.Server
+PackageVersion: 5.14.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: burn
+  Scope: machine
+  InstallerUrl: https://github.com/tgstation/tgstation-server/releases/download/tgstation-server-v5.14.0/tgstation-server-installer.exe
+  InstallerSha256: 97133E67A948155A04D5A06D5128C555D237F92615878A8D818EA54152E51875
+  InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.HostingBundle.6
+  ProductCode: '{D24887FA-3228-4509-B5F3-4E07E349F278}'
+  UnsupportedOSArchitectures:
+  - arm
+  - arm64
+  AppsAndFeaturesEntries:
+  - DisplayName: tgstation-server
+    Publisher: /tg/station 13
+    DisplayVersion: 5.14.0

```

Comment:
```suggestion
```
DisplayVersion should not be used when it's equal to PackageVersion. This causes unnecessary version mapping for the WinGet client. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147. I see this has been used in previous versions as well and should be removed there as well